src/dbnode/persist/fs/msgpack/decoder.go

```diff
@@ -368,8 +368,11 @@ func (dec *Decoder) decodeLogInfo() schema.LogInfo {
 		return emptyLogInfo
 	}
 	var logInfo schema.LogInfo
-	logInfo.Start = dec.decodeVarint()
-	logInfo.Duration = dec.decodeVarint()
+
+	// Deprecated, have to decode anyways for backwards compatibility, but we ignore the values.
```

Comment:
Hm, we can remove the fields and just call `dec.decodeVarint()` twice here possibly.  Thoughts on that approach instead?  Not too bullish on either, but seems a little cleaner to remove the fields from the LogInfo struct.

Reply:
I'd like to keep them in because the structs actually make for good ad-hoc documentation of the file format on disk (we keep all the fields in order)